src/plugins/interpreter/public/index.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { FunctionsRegistry } from './registries';
+
+export const plugin = () => ({
```

Comment:
- [All plugins at Elastic should follow the class convention of defining plugins](https://github.com/elastic/kibana/blob/master/src/core/MIGRATION.md#architecture), it's also useful to implement the `Plugin` interface for that class.
- This plugin needs to export types for it's setup and start contracts.

Reply:
Good to know!
And then things can be imported statically from the top level of the plugin.